examples/boiler_plate/src/cli.c

```diff
@@ -73,7 +73,7 @@ void initialize_cli(int argc, char const *argv[]) {
   if (fio_cli_get("-redis") && strlen(fio_cli_get("-redis"))) {
     FIO_LOG_INFO("* Initializing Redis connection to %s\n",
                  fio_cli_get("-redis"));
-    http_url_s info =
+    fio_url_s info =
```

Comment:
I think we will need to update the function call as well as the type (using [`fio_url_parse`](https://github.com/boazsegev/facil.io/blob/c1306722fd84a23447671619fc76eb8a11a6cb39/lib/facil/fio.h#L866-L915) instead of `http_url_parse`).